configure.ac

```diff
@@ -105,14 +105,14 @@ AC_SEARCH_LIBS([clock_gettime], [rt],,, [$save_LIBS])
 
 AC_MSG_CHECKING([checking for gflags library])
 LIBS="-lgflags $LIBS"
-AC_LINK_IFELSE([AC_LANG_PROGRAM([#include <gflags/gflags.h>], [google::ParseCommandLineFlags(NULL, NULL, true)])], [have_gflags=yes], [have_gflags=no])
+AC_LINK_IFELSE([AC_LANG_PROGRAM([[#include <gflags/gflags.h>], [google::ParseCommandLineFlags(NULL, NULL, true)]])], [have_gflags=yes], [have_gflags=no])
```

Comment:
Is that the right pairing of double square brackets? Shouldn't it be "]], [[" in the middle?